src/main/java/io/nitric/faas/Faas.java

```diff
@@ -52,12 +53,12 @@
  *
```

Comment:
Please update the imports to replace import io.nitric.faas.NitricEvent; with import io.nitric.faas.Trigger;

Reply:
Resolved.